src/openMVG/sfm/pipelines/hierarchical_hyper/hypercluster_test.cpp

```diff
@@ -98,6 +108,8 @@ void generate_sfm_data_and_tracks(openMVG::sfm::SfM_Data & sfm_data, openMVG::tr
     all_view_ids.insert(i);
   }
 
+  sfm_data.intrinsics[0] = std::make_shared<openMVG::cameras::Pinhole_Intrinsic>();
```

Comment:
Is is normal to have a default initialization?
I'm pretty sure you update it later on.